syncer/syncer_test.go

```diff
@@ -80,11 +84,26 @@ func (s *testSyncerSuite) SetUpSuite(c *C) {
 
 	s.resetMaster()
```

Comment:
still need to leave them?
```
s.resetMaster()
s.resetBinlogSyncer()
```
?

Reply:
some tests still use them, but I will remove them eventually in other PRs